src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -0,0 +1,112 @@
+import useTransactionCardStyles from "src/views/activity/components/transaction-card/transaction-card.styles";
+import { ReactComponent as TransferL1Icon } from "src/assets/icons/transfer-l1.svg";
+import { ReactComponent as TransferL2Icon } from "src/assets/icons/transfer-l2.svg";
+import { ReactComponent as ReloadIcon } from "src/assets/icons/reload-card.svg";
+import { ReactComponent as EthToken } from "src/assets/tokens/eth.svg";
+import { ReactComponent as DaiToken } from "src/assets/tokens/dai.svg";
+import FiatAmount from "src/views/shared/fiatAmount/fiat-amount";
+import Typography from "src/views/shared/typography/typography.view";
+import Card from "src/views/shared/card/card.view";
+
+export interface TransactionCardProps {
+  type: "l1" | "l2";
+  id: number;
+  timestamp: number;
+  token: "eth" | "dai";
+  status: "hold" | "initiate" | "process" | "complete" | "error";
+  amount: number;
+}
+
+const TrasnsactionCard = ({ type, timestamp, token, amount, status }: TransactionCardProps) => {
+  const classes = useTransactionCardStyles();
+  const IconsLayer = {
+    l1: TransferL1Icon,
+    l2: TransferL2Icon,
+  };
+  const Icon = status !== "complete" && status !== "error" ? ReloadIcon : IconsLayer[type];
+  const actionText = type === "l1" ? "Transfer to L1" : "Transfer to L2";
+  const statusText = {
+    hold: "On Hold",
+    initiate: "Initiate",
+    process: "Processing",
+    error: "Error",
```

Comment:
Don't we need a status text for `completed`?

Reply:
We don't show the tag when is `completed`